common/src/utils/time.ts

```diff
@@ -1,7 +1,5 @@
-import * as moment from "moment";
-
 export function setNoon(date: string): string {
-  return moment.utc(date.split("T")[0]).set({ hour: 12, minute: 0 }).format();
+  return date.split("T")[0] + "T12:00:00Z";
```

Comment:
This runs 10-20x faster in firefox. I'm seeing huge savings at page load time (for `getAllLayerDays`, the overall time dropped from 1400ms to about 80ms). It also seems to save about 100MB of RAM for rbd.